utxonursery.go

```diff
@@ -1322,7 +1305,7 @@ func (u *utxoNursery) registerPreschoolConf(kid *kidOutput, heightHint uint32) e
 
 	pkScript := kid.signDesc.Output.PkScript
 	confChan, err := u.cfg.Notifier.RegisterConfirmationsNtfn(
-		&txID, pkScript, u.cfg.ConfDepth, heightHint,
+		&txID, pkScript, u.cfg.ConfDepth, kid.broadcastHeight,
```

Comment:
It is, rationale being that for the few users where this currently matters (user with no txindex that has a HTLC output expiring) they will recover with a rescan from genesis. I don't think it will be many cases of this.

Reply:
Is it really worth risking the possibility of doing an entire chain rescan to avoid one db read on startup? It almost seems like we can leave this as is, and just pass the broadcast height in-memory to avoid having two different serialization formats.

Reply:
`broadcastHeight` could just be field on the `kidOutput` that isn't serialized, but set either from the cnct on handoff or from the close summary on restart

Reply:
Just removed the broadcast height from the serialized format, much simpler now!